src/coreclr/inc/bitvector.h

```diff
@@ -32,7 +32,9 @@
 #define UNDEF_ASSERTE
 #endif
 
+#ifndef FEATURE_NATIVEAOT
 #define USE_BITVECTOR 1
+#endif
```

Comment:
This likely should be changed back and `bitvector.cpp` needs to be added to NativeAOT/win-x86 build.

Reply:
BitVector does not seem to be used anywhere else. What is the advantage of using the BITVECTOR wrapper? Can we delete bitvector.h/bitvector.cpp and turn the handful of methods into internal helpers used by x86 GC info decoding only?

Reply:
It is used for `ptrArgTP` which is in turn used in the x86 GC info decoder. I suppose it may be needed for methods with > 64 arguments. For the sake of simplicity I opted to got with `!USE_BITVECTOR` for the bring up but I suppose the `BitVector` is necessary for correctness.

Reply:
Oh, if you mean to merge it with the rest of the extracted code, I guess that would make sense. I'd prefer to do it separately to keep it reviewable.

Reply:
Yes, I would move it together (and maybe renamed it too). It is not a general purpose bitvector. Also, delete the non-BitVector path.

It is fine to do it in a follow up PR.